ui/src/pages/Logs/Logs.tsx

```diff
@@ -93,9 +93,31 @@ export default function Logs() {
     try {
       const allContainers = await fetchAllContainers(ddClient);
       const allContainerLogs = await fetchAllContainerLogs(ddClient, allContainers);
+      // Added new container and filters are applied
+      if (
+        allContainers.length > containers.length &&
+        (filters.allowedContainers.size < containers.length || !filters.stderr || !filters.stdout)
+      ) {
+        setFilters({
+          ...filters,
+          allowedContainers: filters.allowedContainers.add(allContainers[0].Id),
+        });
+      } // If no new containers but filters are applied
+      else if (
+        allContainers.length === containers.length &&
```

Comment:
What if the user deletes one container but starts a brand new container between refreshing the page? The lengths are the same but the containers are different.